.github/workflows/tics-run.yaml

```diff
@@ -0,0 +1,75 @@
+name: TICS QA Analysis
+
+on:
+    schedule:
+        - cron: '0 0 * * 1'  # Runs every Monday at midnight
+    workflow_dispatch:
+
+
+env:
+  apt_deps: >-
+    libpam-dev
+    libglib2.0-dev
+    libpwquality-dev
+    protobuf-compiler
+    protoc-gen-go
+    protoc-gen-go-grpc
+
+jobs:
+    tics:
+        name: TIOBE TICS Framework
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-go@v5
+              with:
+                go-version-file: go.mod
+
+            - name: Update Rust version
+              run: |
+                rustup update stable
+
+            - name: Install dependencies
+              run: |
+                set -eu
+
+                sudo apt-get update
+                sudo DEBIAN_FRONTEND=noninteractive apt-get install -y ${{ env.apt_deps }}
```

Comment:
Do I understand correctly that the `TICSQServer` command below actually runs the tests, and that's why we need to install all the dependencies here?

Reply:
Mmh, I now read https://ticsdocumentation.tiobe.com/latest/docs/#doc=install/install_02_arch.html and https://ticsdocumentation.tiobe.com/latest/docs/admin/admin_A3_qserverref.html#ticsqserver but I still don't really understand what the command does. Could you try running it without building the artifacts before, so that we know if that is actually required?

Reply:
> `[ERROR 5000] Running tool 'go' failed with exit status '256'. Obtained error message: go: go.mod requires go >= 1.24.0 (running go 1.23.12; GOTOOLCHAIN=local)`

That error is caused by `GOTOOLCHAIN=local`. It should not set `GOTOOLCHAIN`, so that Go uses the version specified in `go.mod`.

> but the analysis progressed "normally" as far as I can tell

if it works fine without building the binaries before, IMO we should remove the build steps